cmd/status.go

```diff
@@ -113,7 +111,7 @@ func print(p *status.PackageStatus, w io.Writer) error {
 		for _, change := range p.PendingChanges.Changes {
 			changelogTable = append(changelogTable, formatChangelogEntry(change))
 		}
-		table := tablewriter.NewWriter(os.Stdout)
+		table := tablewriter.NewWriter(w)
```

Comment:
Update to use the same writer as the other table, so it can be tested in `cmd/status_test.go`